cumulusci/cli/cci.py

```diff
@@ -1449,83 +1442,178 @@ def task_info(runtime, task_name):
+
+    def format_help(self, ctx, formatter):
+        """Custom help for `cci task run`"""
+        tasks = RUNTIME.get_available_tasks()
+        plain = RUNTIME.universal_config.cli__plain_output or False
+        task_groups = group_items(tasks)
+        for group, tasks in task_groups.items():
+            data = [["Task", "Description"]]
+            data.extend(sorted(tasks))
+            table = CliTable(data, group, wrap_cols=["Description"])
+            table.echo(plain)
+
+        click.echo("Usage: cci task run <task_name> [TASK_OPTIONS...]\n")
+        click.echo("See above for a complete list of available tasks.")
+        click.echo(
+            "Use "
+            + click.style("cci task info <task_name>", bold=True)
+            + " to get more information about a task and its options."
+        )
 
+    def _collect_task_options(self, new_options, old_options, task_name, task_options):
+        """Merge new style --options with old style -o options.
+
+        Raises:
+            CumulusCIUsageError: if there is an old option which duplicates a new one,
+            or the option doesn't exist for the given task.
+        """
+        # filter out options with no values
+        options = {k: v for k, v in new_options.items() if v is not None}
+        for k, v in old_options:
+            if options.get(k):
+                raise CumulusCIUsageError(
+                    f"Please make sure to specify options only once. Found duplicate option `{k}`."
+                )
+            if k not in task_options:
+                raise CumulusCIUsageError(
+                    f"No option `{k}` found in task {task_name}.\nTo view available task options run: `cci task info {task_name}`"
+                )
+            options[k] = v
+        return options
+
+    def _get_click_options_for_task(self, task_options):
+        """
+        Given a dict of options in a task, constructs and returns the
+        corresponding list of click.Option instances
+        """
+        click_options = [click.Option(["-o"], nargs=2, multiple=True, hidden=True)]
+        for name, properties in task_options.items():
+            # NOTE: When task options aren't explicitly given via the command line
+            # click complains that there are no values for options. We set required=False
+            # to mitigate this error. Task option validation should be performed at the
+            # task level via task._validate_options() or Pydantic models.
+            click_options.append(
+                click.Option(
+                    param_decls=(f"--{name}",),
+                    required=False,  # don't enforce option values in Click
+                    help=properties.get("description", ""),
+                )
+            )
+        return click_options
+
+    def _get_default_command_options(self, is_salesforce_task):
+        click_options = []
+        for opt_name, config in self.global_options.items():
+            if opt_name == "org":
```

Comment:
I think you can just leave `org` out of global_options, and include its help inline below. It doesn't need to be in global_options to be excluded on line 1499, because it already got popped out of kwargs on line 1493